spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -130,7 +140,8 @@ protected Table createTable(String name, Schema schema, PartitionSpec spec) {
     TableOperations ops = ((BaseTable) table).operations();
     TableMetadata meta = ops.current();
     ops.commit(meta, meta.upgradeToFormatVersion(2));
-    if (vectorized) {
+    table.updateProperties().set(TableProperties.DEFAULT_FILE_FORMAT, format).commit();
+    if (format.equals("parquet") && vectorized) {
```

Comment:
This is all about parquet format. We may do this 
```
 if (format.equals("parquet")) {
   if (vectorized) {
      table
          .updateProperties()
          .set(TableProperties.PARQUET_VECTORIZATION_ENABLED, "true")
          .set(
              TableProperties.PARQUET_BATCH_SIZE,
              "4") // split 7 records to two batches to cover more code paths
          .commit();
    } else {
      table.updateProperties().set(TableProperties.PARQUET_VECTORIZATION_ENABLED, "false").commit();
    }
}
```
Also since we added orc and avro, we may set the vectorized property explicitly like we did for `parquet` other than using the default value. The default value may change in the future, which will break the tests.

Reply:
I could have been mistaken, but when I started working on this, vectorized read was only supported for parquet. For this reason, the parameters for the testsuite only has vectorized=true for format=parquet. In any case, the parameters give a fair coverage of both file formats and vectorized/non-vectorized.

I think I understand your concern. If I understand it correctly, your concern is with `TableProperties.ORC_VECTORIZATION_ENABLED` which currently defaults to false. If format=orc, we want to set `TableProperties.ORC_VECTORIZATION_ENABLED` explicitly in case the default changes in the future.

I'll fix this.

Reply:
Fixed.